tests/features/async-components.spec.ts

```diff
@@ -1,17 +1,17 @@
 import { defineAsyncComponent, defineComponent, h, AppConfig } from 'vue'
 
 import { mount, flushPromises } from '../../src'
+import { ComponentPublicInstance } from '@vue/runtime-core'
 
 const config: AppConfig = {
   optionMergeStrategies: {},
   globalProperties: {},
   isCustomElement: (tag: string) => false,
   performance: false,
-  errorHandler: (error: Error) => {
-    if (error.message.match(/Async component failed to load./)) {
+  errorHandler: (error: unknown) => {
```

Comment:
How come `Error` is not the correct type here? I assume `errorHandler` always receives something extending `Error`?

Reply:
I would have thought so but it seems to be [typed to unkown](https://github.com/vuejs/vue-next/blob/6d5b62351248780663d2612a1f483f7ea9f5e5a2/packages/runtime-core/src/apiCreateApp.ts#L58). This may well could be an oversight in the vue next repo

Reply:
This is `unknown` on purpose, it changed during the alphas (IIRC JS can throw other things than error, hence the unknown type)